src/main.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(dead_code)]
+#![allow(clippy::redundant_field_names)]
```

Comment:
Imo it's cleaner to fix the `height: height` issue instead of disabling the check globally

Reply:
I do not find it very consistent to specify some but not all fields, even if clippy prefers this.

Reply:
Agreed